tests/foreman/cli/test_errata.py

```diff
@@ -1382,20 +1384,27 @@ def test_apply_errata_using_default_content_view(errata_host):
     erratum = Host.errata_list({'host': errata_host.hostname, 'search': f'id = {REAL_0_ERRATA_ID}'})
     assert len(erratum) == 1
     assert erratum[0]['installable'] == 'true'
-    # note time for later wait_for_tasks include 2 mins margin of safety.
-    timestamp = (datetime.utcnow() - timedelta(minutes=2)).strftime(TIMESTAMP_FMT)
     # Update errata from Library, i.e. Default CV
-    errata_host.run(f'yum -y update --advisory {REAL_0_ERRATA_ID}')
-    # Wait for upload profile event (in case Satellite system slow)
+    result = JobInvocation.create(
+        {
+            'feature': 'katello_errata_install',
+            'search-query': f'name = {errata_host.hostname}',
+            'inputs': f'errata=\'{REAL_0_ERRATA_ID}\'',
```

Comment:
Use double quotes instead of string escapes

Reply:
No can do. 
The `cmd` in `ssh.command()` gets constructed like this:
```
'LANG=en_US.UTF-8  hammer -v -u admin -p changeme --output=csv job-invocation create --feature="katello_errata_install" --search-query="name = <host>" --inputs="errata="RHBA-2021:1314"" --organization-id="4" '
```
^^The `inputs` arg gets messed up.

Reply:
Yes can do.